src/libsystemd/sd-bus/bus-common-errors.c

```diff
@@ -60,8 +60,7 @@ BUS_ERROR_MAP_ELF_REGISTER const sd_bus_error_map bus_common_errors[] = {
         SD_BUS_ERROR_MAP(BUS_ERROR_SLEEP_VERB_NOT_SUPPORTED,     EOPNOTSUPP),
         SD_BUS_ERROR_MAP(BUS_ERROR_SESSION_BUSY,                 EBUSY),
         SD_BUS_ERROR_MAP(BUS_ERROR_NOT_YOUR_DEVICE,              EPERM),
-        /* needs to be EOPNOTSUPP for proper handling in callers of logind_schedule_shutdown() */
-        SD_BUS_ERROR_MAP(BUS_ERROR_DESIGNATED_MAINTENANCE_TIME_NOT_SCHEDULED, EOPNOTSUPP),
+        SD_BUS_ERROR_MAP(BUS_ERROR_DESIGNATED_MAINTENANCE_TIME_NOT_SCHEDULED, EBADSLT),
```

Comment:
hmm, you changed this now, but that means we'll no reboot at all if no maintenance time is configured, right?

Reply:
Nope. Previously `EOPNOTSUPP` was used to _disable_ fallback logic in systemctl. So none of the commits change the behavior regarding unset designated maintenance window

Reply:
indeed.